resources/cygnus-translators/cygnus_translator_pre0.10.0_to_0.10.0_ckan.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+# -*- coding: latin-1 -*-
+
+# Author: frb
+
+# Imports
+import sys
+import requests
+import json
+import pprint
+
+# Show the usage
+if len(sys.argv) < 7:
+   print('Usage: cygnus_translator_pre0.10.0_to_0.10.0_hdfs.sh ckan_host ckan_port api_key org_name attr_persistence null_value backup')
+   print('where ckan_host')
+   print('      ckan_port')
+   print('      api_key')
+   print('      org_name')
+   print('      attr_persistence')
+   print('      null_value : string to be inserted as null value, use the keyword \"empty\" for an empty value')
+   print('      backup     : either true or false')
+   sys.exit(0)
+
+# Input parameters
```

Comment:
There is a nice library that I discover recently to do argument parsing. You can have a look to an example here: https://github.com/telefonicaid/fiware-orion/blob/develop/scripts/managedb/csubs_control/csub_dups_checker.py#L47

If you find it useful but you are in a rush to close this PR, I'd suggest to include a FIXME about it.